client/src/App.jsx

```diff
@@ -2,45 +2,50 @@ import React from 'react';
 import AddCommentForm from './AddCommentForm.jsx';
 import CommentList from './CommentList.jsx';
 import axios from 'axios';
+import About from './About.jsx';
 
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      comment: [],
+      comments: [],
     };
     this.getComment = this.getComment.bind(this);
     this.addComment = this.addComment.bind(this);
   }
 
+  // testing with a recipe ID
   componentDidMount() {
-    this.getComment();
+    this.getComment(2);
   }
 
 
-  getComment() {
-    axios.get('/recipe')
-    .then(recipes => this.setState({ comment: recipes.data }))
-    .then(recipe => console.log(this.state.comment))
-    .catch(err => console.log('Error getting comments', err));
-    
+  getComment(id) {
+    axios.get(`/recipe/${id}`)
+    .then(comment => this.setState({comments: comment.data[0].comments}))
```

Comment:
I'm looking at the format of the data you receive when making an API call and it is very confusing; it looks like a get request to the API gives you an array consisting of a single object which consists an array of comments. The comment key is unnecessary and creates confusion. Instead, your data should simply be an array of objects since your route (which you should change to recipes/:id/reviews or recipes/:id/comments) would already make it clear to anyone reading your code that you are going to be receiving a collection of comments.